lib/client.js

```diff
@@ -231,7 +231,7 @@ class BPClient extends Client {
   /*
    * Until bcoin, bcash and hsd return SPV flags in `info` we need to test
    */
-  async isSPV() {
+  async setSPV() {
     try {
       await this.node.getBlock(0);
       this.SPV = false;
```

Comment:
should this also be `this.isSPV` now that the name is free? This will make code like `if (client.isSPV) ...` read nicer I think.

Reply:
Hm I don't think so. `SPV` sounds like a `bool`, `setSPV` sounds like something is happening and `isSPV` sounds like a question. SO I think it makes sense like this.

`SPV` is the actual thing, `setSPV` sets it and `isSPV` gets it (in recent-blocks).

Your preference though, not gunna pick this battle :-)

Reply:
Yeah, actually `SPV` sounds like the actual thing to me more than a bool, so I would expect it to return some SPV object. In fact, that's how you updated recent-blocks already, with `isSPV` set to a bool. This would mirror behavior elsewhere in bcoin (e.g. [`Input.isInput`](https://github.com/bcoin-org/bcoin/blob/85ed59c842f2aa4c1a8154d74a9cad7696cdfee3/lib/primitives/input.js#L519)) or normal JavaScript (e.g. `Array.isArray`)

Reply:
Cool! updating